BuildGuide.md

```diff
@@ -77,13 +77,12 @@
       cd ~/
       git clone https://github.com/Start9Labs/embassy-os.git
```

Comment:
This assumes that the Start9Labs/embassy-os#252 pull request is merged. That is why I use:
`git clone -b BuildGuide https://github.com/k0gen/embassy-os.git`
To not confuse user following this guide.